record-two/index.html.in

```diff
@@ -35,7 +35,18 @@
             <div uib-accordion-group class="related-table-heading" id="rt-heading-{{relatedRef.displayname}}" ng-repeat="relatedRef in relatedReferences track by $index"
                 ng-if="($index === 0 || tableModels[$index-1].hasLoaded) && tableModels[$index].rowValues.length > 0"
                 heading="{{tableModels[$index].open ? '-' : '+'}} {{relatedRef.displayname}} ({{tableModels[$index].rowValues.length}})" is-open="tableModels[$index].open">
-                <record-table class="related-table" id="rt-{{relatedRef.displayname}}" vm="tableModels[$index]"></record-table>
+                <!-- for different elements inside the accordion -->
+                <div ng-switch on="relatedRef.display.type">
+                    <div ng-switch-when="markdown">
+                        <span ng-bind-html="page.content"></span>
+                        <span ng-if="page.hasNext" ng-controller="RecordController as ctrl" ng-click="ctrl.toRecordSet(relatedRef)">Click here to access more records</span>
+                    </div>
+                    <!-- ng-switch-when="table" is also the default case -->
+                    <div ng-switch-default>
+                        <record-table class="related-table" id="rt-{{relatedRef.displayname}}" vm="tableModels[$index]"></record-table>
+                        <a ng-controller="RecordController as ctrl" ng-click="ctrl.toRecordSet(relatedRef)">Click here to access more records</a>
```

Comment:
That was for testing. This PR isn't technically ready yet, I still need to flesh out the test cases.